### PR TITLE
App Title Change

### PR DESCRIPTION
**Pull request by mwernimont**
Before making a pull request
----------------------------
First . . .
- [ ] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [x] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [x] Chrome
- [x] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
Changed the apps title by request, kept the build description as it does help me see what version I am on.

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,5 @@
-VUE_APP_TITLE=National Integrated Water Availability Assessments
+VUE_APP_TITLE=National Integrated Water Availability Assessments: Water Availability Mapper
 
-VUE_APP_TITLE_SUFFIX=-Concept Map
+VUE_APP_TITLE_SUFFIX=
```
Comment (mhines-usgs) [inline .env:3]: wondering if this suffix is baked into any other places in the app, if so, we might want to just remove it?

Comment (mwernimont) [reply .env:3]: As far as I can tell it was only for the title?  So I can see removing it being fine.  Do you see anything weird without it?

Comment (mwernimont) [reply .env:3]: Removing it locally had no noticeable issues and console is clean.  Meh, can always puyt it back if we find soemthing later

**Comment by mhines-usgs** [comment]
I meant like removing the references too it, too 
https://github.com/usgs-makerspace/wbeep-viz/search?q=VUE_APP_TITLE_SUFFIX

**Comment by mhines-usgs** [comment]
looks goooood

**Pull request merged**